node/plugin/config.go

```diff
@@ -14,6 +14,31 @@ import (
 var (
 	/* Required Flags */
 
+	HostnameFlag = cli.StringFlag{
```

Comment:
All these 3 flags are already encoded in the existing `SocketFlag`, which you may parse. So there is no need to redefine them here again.